src/main/java/frc/robot/subsystems/led/Led2023.java

```diff
@@ -663,30 +770,46 @@ public void setAlternateColorsUp(COLORS_467 colorOne, COLORS_467 colorTwo, Color
 
   public void setOneThird(COLORS_467 color, int preSet) {
     // t=1, 2, or 3. sets top 1/3, mid 1/3, or lower 1/3
-    double start;
-    double end;
-    set(COLORS_467.Black);
+    int start;
+    int end;
+
     if (preSet == 1) {
-      start = 0;
-      end = Math.ceil(RobotConstants.get().led2023LedCount() / 3);
+      start = (int) topStart;
+      end = (int) topEnd;
     } else if (preSet == 2) {
-      start = Math.floor(RobotConstants.get().led2023LedCount() / 3);
-      end =
-          Math.ceil(
-              RobotConstants.get().led2023LedCount()
-                  - (RobotConstants.get().led2023LedCount() / 3));
+      start = (int) midStart;
+      end = (int) midEnd;
 
     } else {
-      start =
-          Math.floor(
-              RobotConstants.get().led2023LedCount()
-                  - (RobotConstants.get().led2023LedCount() / 3));
-      end = Math.ceil(RobotConstants.get().led2023LedCount());
+      start = (int) bottomStart;
+      end = (int) bottomEnd;
     }
-    for (int i = (int) start; i < end; i++) {
+    for (int i = start; i < end; i++) {
       ledStrip.setLED(i, color.getColor());
     }
 
     ledStrip.update();
   }
+
+  public void victoryLeds(COLORS_467 fgColor, COLORS_467 bgColor) {
+    if (victoryTimer.hasElapsed(0.9)) {
+      victoryTimer.reset();
+    } else if (victoryTimer.hasElapsed(0.4)) {
+      for (int i = 0; i < RobotConstants.get().led2023LedCount(); i++) {
+        for (int brightness = 0; brightness <= 256; brightness++) {
```

Comment:
I think doing this all in a single call will freeze the rest of the robot's functions, so it would be better to increment this each time it's called.